framework/src/timesteppers/FunctionDT.C

```diff
@@ -20,7 +20,7 @@ template<>
 InputParameters validParams<FunctionDT>()
 {
   InputParameters params = validParams<TimeStepper>();
-  params.addParam<std::vector<Real> >("time_t", "The values of t");
+  params.addRequiredParam<std::vector<Real> >("time_t", "The values of t");
   params.addParam<std::vector<Real> >("time_dt", "The values of dt");
```

Comment:
But function implies that there is independent and dependent variable, while when prescribing time points is not the case. I can also see somebody wanting to prescribe the points from an external file... So I lean towards a separate time stepper